docs/release_notes.md

```diff
@@ -1,5 +1,46 @@
 # Release Notes
 
+## Version 1.4.0
+
+### Core
+
+- Implemented a new solver to make muscles behave more compact: AdnGlue solver.
+- Implemented Soft Constraints.
+- Added parallelization to Distance constraints.
+- Added parallelization to Shape Preservation constraints.
+- Added Hard Constraints to theAdnFat solver.
```

Comment:
```suggestion
- Added Hard Constraints to the AdnFat solver.
```

Reply:
weird that Google Doc did not highlight this one! thank you, fixed now